src/main/java/com/synopsys/integration/detect/workflow/event/Event.java

```diff
@@ -68,4 +69,5 @@
     public static final EventType<NameVersion> ProjectNameVersionChosen = new EventType<>(NameVersion.class);
     public static final EventType<ExecutedExecutable> Executable = new EventType<>(ExecutedExecutable.class);
     public static final EventType<UnrecognizedPaths> UnrecognizedPaths = new EventType<>(UnrecognizedPaths.class);
+    public static final EventType<Map> PropertyValuesCollected = new EventType<>(Map.class);
```

Comment:
I do not like the type being a Map as we lose the types of the Map. I'd prefer we created and published a wrapper object.

Reply:
Fair